sdk/key/soft_key.go

```diff
@@ -283,12 +283,12 @@ func (m *SoftKey) Save(p string) error {
 	return os.WriteFile(p, []byte(m.PrivKeyHex()), constants.UserOnlyWriteReadPerms)
 }
 
-func (m *SoftKey) P(networkHRP string) (string, error) {
-	return address.Format("P", networkHRP, m.privKey.PublicKey().Address().Bytes())
+func (m *SoftKey) P(network network.Network) (string, error) {
```

Comment:
we already have this P object, why do we need another keychain P chain function?